.NET/Microsoft.Recognizers.Text.DateTime/Parsers/CJK/BaseCJKTimePeriodParser.cs

```diff
@@ -106,10 +106,22 @@ private DateTimeResolutionResult ParseTimeOfDay(string text, DateObject referenc
                 return new DateTimeResolutionResult();
             }
 
+            // Add "early"/"late" Mod
+            if (endHour == beginHour + 2 && (beginHour == Constants.HalfDayHourCount - 4 || beginHour == Constants.HalfDayHourCount))
```

Comment:
This should also be a common method. When extracting it, please also define constants for '2' and '4'.

Reply:
also here the standard implementation follows different implementation as described above

Reply:
But the same as in other languages, the definitions of what early/late mean or the lengths of the ranges should be uniform.

Reply:
used value defined in Constants